### ISSUE TITLE
Confused about the use of layernormalization in your Imagenet training code.

### ISSUE THREAD

**Issue by WANGXinuuu1** [ZK-Zhou/spikformer#14]
Thanks for your kind sharing for your codes. You did a really good and meaningful research work in the field of SNN.

I just confused about # norm_layer=nn.LayerNorm # in your code of training Imagenet.

But in your paper, I found the state "We empircally find that the layer normalization (LN) does not apply to SNNs, so we use batch normalization (BN) instead." It seems a little bit inconsistent.

Many thanks!

Xin Wang
